CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changes by Version
 ==================
 
+2.20.2 (2019-12-20)
+-------------------
+
+- Support custom HTTP headers when reporting spans over HTTP ([#479](https://github.com/jaegertracing/jaeger-client-go/pull/479)) -- Albert Teoh
```

Comment:
this is a new feature, please bump to 2.21

Reply:
also, there are other commits besides this one, changelog must include everything